src/Sarif/Sarif.csproj

```diff
@@ -38,8 +38,6 @@
     <Content Include="Schemata\Sarif.schema.json" />
 
     <!-- Link=false: Don't display this item in Solution Explorer. -->
-    <Content Include="..\ReleaseHistory.md"
```

Comment:
1. If you remove it from the project file, it won't automatically be added to the NuGet package. That's why it's here. I suppose the comment on the line above should have mentioned that.
2. If you do decide to remove it, remove that comment.

Reply:
Good catch, not sure how this got stripped. Restored it and added a comment about the line adding the referenced file to the NuGet, to try to ensure it remains in future. I suspect some random act in the IDE scrubbed. 

---
In reply to: [205979938](https://github.com/Microsoft/sarif-sdk/pull/974#discussion_r205979938) [](ancestors = 205979938)